docs/en/datasets/recog.md

```diff
@@ -79,6 +79,13 @@
 │   │   ├── annotations
 │   │   ├── train_label.txt
 │   │   ├── test_label.txt
+│   ├── ILST
```

Comment:
It seems you didn't `mkdir ILST` in the doc. Please check if the directory name is correct. Also, `ILST` or `ilst` (since I saw `ilst` on the `shared_data` path)?

Reply:
My bad.  I think I'll just use `IIIT-ILST` here.